README.md

```diff
@@ -133,16 +137,15 @@ First we get the CA cert for the front proxy, saving it to a file `ca.crt`
 
 Now we create a new kubeconfig which references the `ca.crt`
 
-    kubectl --kubeconfig=admin.kubeconfig config set-cluster base --server https://<externalHostname>:443 --certificate-authority=ca.crt
-    kubectl --kubeconfig=admin.kubeconfig config set-cluster root --server https://<externalHostname>:443/clusters/root --certificate-authority=ca.crt
+    kubectl --kubeconfig=admin.kubeconfig config set-cluster base --server https://$KCP_EXTERNAL_HOSTNAME:443 --certificate-authority=ca.crt
+    kubectl --kubeconfig=admin.kubeconfig config set-cluster root --server https://$KCP_EXTERNAL_HOSTNAME:443/clusters/root --certificate-authority=ca.crt
 
 ### Create client-cert credentials
 
 Now we must add credentials to the kubeconfig, so requests to the front-proxy may be authenticated.
 
 One way to do this is to create a client certificate with a cert-manager `Certificate`:
-
-    $ cat admin-client-cert.yaml
```

Comment:
Question: Why (here and below) remove the `$`? They are usually known as "this is a command you need to type in" in shell snippets.

Reply:
![image](https://github.com/user-attachments/assets/4426c196-9213-47cb-9e12-ad955d335996)
This Markdown document currently displays it as a whole block, so when you want to copy, you would need to remove `$` manually.
Keeping `$` would be also an option, but I would think of adding more context to some specific commands that we want to highlight.

Reply:
ah, that makes sense. You can copy it at once this way.